examples/jquery/index.html

```diff
@@ -21,15 +21,7 @@
     <div class="hidden" id="logged-in">
         <div>
             <div>
-                <div>
-                    Button Clicks: <span id="js-button-clicks"></span>
-                </div>
-                <button id="js-decrement">
-                    Decrement trait
-                </button>
-                <button id="js-increment">
-                    Increment trait
-                </button>
+                Button Clicks: <span id="js-button-clicks"></span>
```

Comment:
@kyle-ssg this was the one bit I wasn't sure about. Should this whole div be removed?